src/main/java/org/mule/extension/http/internal/listener/HttpListener.java

```diff
@@ -111,6 +111,9 @@
   private static final HttpListenerErrorResponseBuilder NULL_ERROR_RESPONSE = new HttpListenerErrorResponseBuilder();
 
   @Inject
+  private TransformationService transformationService;
+
+  @Inject
   private MuleContext muleContext;
```

Comment:
MuleContext is not used for that purpose in that method. It Is used for getting the default encoding.